scripts/pre_commit_linter.py

```diff
@@ -1140,7 +1140,7 @@ def _lint_py_files(
 
 
 class LintChecksManager(object):
```

Comment:
Since we don't want this to be instantiated directly, we want to have this be an abstract class.

Reply:
Done